CONTRIBUTING.md

```diff
@@ -168,7 +168,6 @@ Here are the trigger phrases for individual checks:
 * `/test-windows-e2e`: Windows IPv4 e2e tests
 * `/test-windows-conformance`: Windows IPv4 conformance tests
 * `/test-windows-networkpolicy`: Windows IPv4 networkpolicy tests
-* `/test-windows-proxyall-e2e`: Windows IPv4 e2e tests with proxyAll enabled
```

Comment:
I may missed the history, why is this no longer needed?

Reply:
proxyAll is enabled by default on Windows. Our CI test always uses proxyAll configurations